Reconstruction/PFA/Pandora/GaudiPandora/include/Utility.h

```diff
@@ -2,5 +2,17 @@
 #define MYUTILITY 1
 
 #include <sstream>
+#include "DD4hep/Detector.h"
+#include "DD4hep/DD4hepUnits.h"
+#include "DD4hep/DetType.h"
+#include "DDRec/DetectorData.h"
+#include "DD4hep/DetectorSelector.h"
 std::string Convert (float number);
+dd4hep::rec::LayeredCalorimeterData * getExtension(unsigned int includeFlag, unsigned int excludeFlag) ;
```

Comment:
The names of the functions may be conflict with others. Consider to add namespace in your utility.

Reply:
Hi @mirguest 
Thanks for your comments! I have updated it, please check it!

Best,
Wenxing